packages/services/src/requests.ts

```diff
@@ -39,10 +39,10 @@ export class RequestHandler {
    * @returns a promise that resolves with the server response on success or
    * an error dialog result in cases of failure.
    */
-  static async makeGetRequest(
+  static async makeGetRequest<T = any>(
```

Comment:
If this is the only change here we should just update all the functions in this class to match